tools/workflow/clean_aws_rsc.go

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package main
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	"log"
+	"time"
+	"strings"
+)
+const (
+	containLbString = "aoc-lb"
+	pastDayDelete = 5
+)
+
+func main() {
+
+	log.Printf("Beging terminating EC2 Instances")
+	terminateEc2InstancesError := terminateEc2Instances()
+	if (terminateEc2InstancesError != nil){
+		log.Fatalf("Exit terminating EC2 Instances")
+	}
+
+	log.Printf("Begin destroy Load Balancer resources")
+	destroyLoadBalancerError := destroyLoadBalancerResource()
+
+	if (destroyLoadBalancerError != nil){
+		log.Fatalf("Exit destroy Load Balancer Resources")
+	}
+
+	log.Printf("Finish destroy AWS resources")
+}
+
+func terminateEc2Instances()(error){
+	// set up aws go sdk ec2 client
+	testSession := session.Must(
+		session.NewSessionWithOptions(
+			session.Options{
+				SharedConfigState: session.SharedConfigEnable,
+			}))
+
+	ec2client := ec2.New(testSession)
+
+	// Get list of instance
+	//State filter
+	instanceStateFilter := ec2.Filter{Name: aws.String("instance-state-name"), Values: []*string{aws.String("running")}}
+
+	//Tag filter
+	instanceTagFilter := ec2.Filter{Name: aws.String("tag:Name"), Values: []*string{aws.String("Integ-test-Sample-App"), aws.String("Integ-test-aoc")}}
+
+	//get instances to delete
+	var deleteInstanceIds []*string
+	var nextToken *string
+	for {
+		describeInstancesInput := ec2.DescribeInstancesInput{Filters: []*ec2.Filter{&instanceStateFilter, &instanceTagFilter}, NextToken: nextToken}
+		describeInstancesOutput, describeInstanceErr := ec2client.DescribeInstances(&describeInstancesInput)
```

Comment:
nitpick: Using and reassigning the same `err` variable is fine. Didn't need to create different variables for each.

Reply:
Its weird for me that went I ran with the same ```err``` variable, it shows me the error of not able to assigning multiple values to err var. That's why I changed to this to be able to run locally

Reply:
You should be able to assign (=) multiple times. You just won't be able to declare (:=) more than once unless another variable is being declared at the same time.

```go
a, err := someFunc()
_, err = otherFunc() // := would be invalid
b, err := anotherFunc() // b is being declared
```

Reply:
Make sense. Another thing to learn in go. Fixed it